diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index c737c93..c23be5e 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -337,7 +337,7 @@ module.exports = {
       .VirtualViewMode;
   },
   // #endregion
-} as ReactNativePublicAPI;
+}; // as ReactNativePublicAPI;
 
 if (__DEV__) {
   /* $FlowFixMe[prop-missing] This is intentional: Flow will error when
